nilimamaka/SpringBoot/MultipleDBConnection/src/main/java/com/example/multipledbconnection/fileDetails/model/FileDetails.java

```diff
@@ -0,0 +1,30 @@
+package com.example.multipledbconnection.fileDetails.model;
+
+import com.example.multipledbconnection.fileDetails.enums.FDStatus;
+import lombok.*;
+
+import javax.persistence.*;
+
+
+@Entity
+@Data
+@Table (name="fileDetails")
```

Comment:
Make sure whether you just want to change the name your entity table or also during the query for entity.

I prefer @Entity(name="file_details")

Camel case is not configured good way for tables. I will suggest either snake case or snake case with capital letters.